File: testing/highlight-samples/sample.ts
// This is a comment
import { User } from './models'

interface Config {
    name: string
    count: number
    active: boolean
}

const greeting: string = "hello world"
const age: number = 42
const enabled = true
const nothing = null

/* Block comment */
function greet(user: User): string {
    return `Hi, ${user.name}!`
}

File: testing/highlight-samples/sample.kt
// This is a comment
import kotlin.math.PI

class User(val name: String, val age: Int = 42) {
    val active: Boolean = true

    fun greet(): String {
        return "Hi, $name!"
    }

    /* Block comment */
}

val count = 3.14
val nothing = null
